cpp/test/neighbors/ann_ivf_pq.cuh

```diff
@@ -879,7 +879,7 @@ inline auto enum_variety_ip() -> test_cases_t
         // InnerProduct score is signed,
         // thus we're forced to used signed 8-bit representation,
         // thus we have one bit less precision
-        y.min_recall = y.min_recall.value() * 0.90;
+        y.min_recall = y.min_recall.value() * 0.88;
```

Comment:
Thanks for opening this PR. Can you also open a GIthub issue for this so that we don' tlose sight of it?

Reply:
https://github.com/rapidsai/cuvs/issues/581